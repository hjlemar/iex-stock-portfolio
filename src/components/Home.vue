<template>
  <v-container id="home">
    <v-layout row>
      <v-flex xs6 mx-auto>
        <app-search></app-search>
      </v-flex>
    </v-layout>
    <v-layout row>
      <v-flex mx-auto>
         <router-view/>
      </v-flex>
    </v-layout>
  </v-container>
</template>

<script>
import Search from '@/components/Search';

export default {
  components: {
    appSearch: Search,
  },
};
</script>

<style>

</style>

