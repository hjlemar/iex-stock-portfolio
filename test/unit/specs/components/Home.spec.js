import { shallow, createLocalVue } from '@vue/test-utils';
import Home from '@/components/Home';
import Vuetify from 'vuetify';

const localVue = createLocalVue();
localVue.use(Vuetify);

describe('Home', () => {
  it('should render component and match snapshot', () => {
    const wrapper = shallow(Home, { store: {}, localVue });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
