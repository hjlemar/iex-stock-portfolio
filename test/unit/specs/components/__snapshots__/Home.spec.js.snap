// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render component and match snapshot 1`] = `undefined`;
